shared/components/Schedule/index.scss

```diff
@@ -1,102 +1,175 @@
+
+  &__date {
+    display: inline-block;
+    font-size: 20px;
+    margin: 0;
+    line-height: 29px;
+    font-weight: normal;
   }
-  &__subtext {
-    margin-top: 30px;
+
+  &__list {
+    margin: 34px 0 30px;
+    border-left: 3px dashed $grey;
+
+    @media (min-width: $tablet-portrait-width) {
+      border-left: 0;
+      position: relative;
+
+      &:after {
+        content: '';
+        position: absolute;
+        left: $left-column-width;
+        bottom: 0;
+        height: calc(100% - 4px);
+        border-left: 3px dashed $grey;
+      }
+    }
   }
-  &__caterer {
-    font-weight: bold;
-    text-decoration: underline;
-    color: $grey;
+
+  &__info {
+    margin-bottom: 0;
   }
-  &__item--time {
-    margin: 70px 0px 20px 0px;
-    position: relative;
-    &:before {
-      content: attr(data-time);
-      top: -30px;
-      text-align: right;
-      color: #4a4a4a;
-      position: absolute;
+}
+
+.ScheduleItem {
+  display: flex;
+  flex-wrap: wrap;
+
+  &__header {
+    width: 100%;
+    text-align: left;
+    padding-left: 17px;
+
+    &__time {
+      display: block;
+      font-size: 16px;
+      line-height: 21px;
+      font-weight: bold;
+      position: relative;
+
+      &::after {
+        position: absolute;
+        content: '';
+        display: inline-block;
+        left: -26px;
+        top: 3px;
+        width: 16px;
+        height: 16px;
+        border-radius: 8px;
+        background-color: $grey;
+
+        @media (min-width: $tablet-portrait-width) {
+          left: auto;
+          right: -26px;
+        }
+      }
     }
-    &:after {
-      margin-top: -12px;
-      content: "";
-      left: -48px;
-      width: 15px;
-      height: 15px;
-      border-radius: 50%;
-      background: #4a4a4a;
-      position: absolute;
+    &__title {
+      font-size: 20px;
+      line-height: 29px;
+      margin-top: 10px;
+      margin-bottom: 30px;
     }
-  }
-  &__item--first {
-    margin-top: 0px;
-  }
-  &__item--last {
-    margin-bottom: -30px;
-  }
-  &__item {
-    margin: 40px 0px;
-    &:after {
-      margin-top: 2px;
-      content: "";
-      left: -8px;
-      width: 15px;
-      height: 15px;
-      border-radius: 50%;
-      background: #4a4a4a;
-      position: absolute;
+
+    @media (min-width: $tablet-portrait-width) {
+      width: $left-column-width;
+      text-align: right;
+      padding-right: 17px;
     }
   }
-}
 
+  &__details {
+    width: 100%;
+    padding-left: 17px;
+
+    &__description {
+      font-size: 16px;
+      line-height: 1.31;
+      margin-top: 0;
+      margin-bottom: 21px;
+      display: block;
+    }
 
-.Schedule {
-  @media (min-width: $tablet-portrait-width) {
-    padding-left: 0px;
-    padding-top: 40px;
     &__list {
-      left: 180px;
+      list-style-type: none;
+      padding: 0;
     }
-    &__header {
-      margin: 0px 0px 20px 0px;
+
+    @media (min-width: $tablet-portrait-width) {
+      width: calc(100% - 200px);
```

Comment:
You can use `margin-left: 200px;` instead. I think `calc` should only be used when really desperate